src/redux/configureStore.js

```diff
@@ -0,0 +1,14 @@
+import { createStore, combineReducers, applyMiddleware } from 'redux';
+import thunk from 'redux-thunk';
+import booksReducer from './books/books';
+
+const reducer = combineReducers({
+  booksReducer,
+});
+
+const store = createStore(
+  reducer,
+  applyMiddleware(thunk),
+);
+
+export default store;
```

Comment:
- [ ] Your project isn't running locally, kindly make sure you installed `redux-thunk` successfully.

![rr27](https://user-images.githubusercontent.com/55635977/135509267-d9e18779-bd74-4083-adc5-38b69e26d731.png)

![rr28](https://user-images.githubusercontent.com/55635977/135509286-b3e48d6c-5499-4140-b920-2f64952078c5.png)

Reply:
This issue is resolved. Great job! :+1: :clap: